kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -350,6 +467,15 @@ private static Map<String, Variant<?>> createWPAWPA2Settings(NetworkProperties p
         return settings;
     }
 
+    private static Map<String, Variant<?>> createWPA2WPA3EnterpriseSettings(NetworkProperties props, String deviceId,
+            String propMode) {
+        Map<String, Variant<?>> settings = new HashMap<>();
+
+        settings.put("key-mgmt", new Variant<>("wpa-eap"));
```

Comment:
Make Sonar stop whining here by creating a constant for storing the `key-mgmt` key. Look at the already existing constants like [NM_SETTINGS_IPV4_METHOD](https://github.com/eclipse/kura/blob/f99573b3fe5efd6ff7ea2411e54982f33d3cce8e/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java#L44) for refernce